### PR TITLE
"Application Structure with Components" and "Using Other Components" sections - angular style guide update

### PR DESCRIPTION
**Pull request by alejochang**
- Updated "Application Structure with Components" and "Using Other Components" sections markdowns inline code to be closer to the angular 2 style guide. #519
- Deleted unused file.

### CODE REVIEW

File: handout/components/structural_directives.md [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-
-<div *ngFor="let hero of heroes">{{ hero }}</div>
-
-<div [ngSwitch]="status">
-  <template *ngSwitchCase="'in-mission'">In Mission</template>
-  <template *ngSwitchCase="'ready'">Ready</template>
-  <template *ngSwitchDefault>Unknown</template>
-</div>
-```
-
```
Comment (ggprod) [inline handout/components/structural_directives.md]: why remove this file related to a different section (structural directives)?

Comment (alejochang) [inline handout/components/structural_directives.md]: The file used in the book, is located under the `directives` folder:
`handout/directives/structural_directives`
This one under `components` folder, had no reference in the main markdown file: `SUMMARY.md`

Comment (ggprod) [inline handout/components/structural_directives.md]: ok

**Comment by bennett000** [comment]
:shipit:

**Pull request merged**